build.gradle

```diff
@@ -79,6 +80,12 @@ tasks.create("currentVersion") {
     }
 }
 
+tasks.create("setEnterpriseId") {
```

Comment:
how do you run it
when is it run

Reply:
`./gradlew setEnterpriseId` - but it doesn't really matter.
just more convenient if you build locally and want to add a pre-built task 🤷🏻‍♂️